modules/nw-osp-specify-floating-ip.adoc

```diff
@@ -0,0 +1,64 @@
+// Modules included in the following assemblies:
+//
+// * networking/load-balancing-openstack.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-osp-specify-floating-ip_{context}"]
+= Specifying a floating IP address in the Ingress Controller
+
+By default, a floating IP address gets randomly assigned to your {product-title} cluster on {rh-openstack-first} upon deployment. This floating IP address is associated with your Ingress port.
+
+You might want to pre-create a floating IP address before updating your DNS records and cluster deployment. In this situation, you can define a floating IP address to the Ingress Controller. You can do this regardless of whether you are using Octavia or a user-managed cluster.
```

Comment:
@EmilienM , is the last statement true? "You can do this regardless of whether you are using Octavia or a user-managed cluster."

Reply:
I don't understand what "a user-managed cluster" means. I think the right wording would be "whether you are using Octavia or another load-balancer backend in your cluster", which would be wrong because this field is only understood (as documented) by the openstack cloud provider, which only supports Octavia now.

Reply:
I guess "user-managed cluster" was supposed to be "user-managed load balancer", which is a phrase that is used elsewhere in the "Load balancing on OpenStack" documentation.  

And, yeah, I guess that a user-managed load balancer is the same thing as what you are saying ("another load-balancer backend in your cluster").